instrumentation/metric/histogram_race.go

```diff
@@ -18,16 +18,16 @@ type Histogram struct {
 type histogramExport struct {
 }
 
-func newHistogram(name string, max int64, n int) *Histogram {
+func newHistogram(name string, pNname string, max int64, n int) *Histogram {
 	return &Histogram{name: name}
 }
 
 func (h *Histogram) Name() string {
 	return h.name
 }
 
-func (h Histogram) Export() exportedMetric {
-	return &histogramExport{}
+func (h Histogram) Export() interface{} {
```

Comment:
this does not bother us, just curious why this did not work

Reply:
this is the races build version and i forgot to write the function so it didn't compile. very annoying this histogram races :)